2/src/components/ListItem.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+			<li>
+				<div v-if="displayTodo">
+					{{ todoString }}
+					<button class="editButton" v-on:click="displayTodo=false">Edit</button>
+					<button class="deleteButton" v-on:click="$emit('delete-todo')">Delete</button>
+				</div>
+				<div v-else>
+					<input placeholder="Enter todo" v-model="inputValue"/>
+					<button class="saveButton" v-on:click="saveTodo">Save</button>
+					<button class="cancelButton" v-on:click="displayTodo=true">Cancel</button>
+				</div>
+			</li>
+</template>
```

Comment:
In this file you use tabs and in the file above you use spaces for indentation. Your linter should actually catch that.